linode/manager
# refactor: [M3-6897] - MUI v5 Migration - `SRC > Components > TableFooter`

## Description 📝

- Component migration for the `<TableFooter />` component

## Major Changes 🔄
- Only use named exports ✏️
- Add Storybook story 📄
- Moved component out of `core` folder to match other components 🗂️

## Preview 📷
![Screenshot 2023-07-17 at 11 09 32 PM](https://github.com/linode/manager/assets/115251059/50909749-ee57-4cc6-aef6-77f622d85653)

## How to test 🧪
- Look at the storybook story
- Verify the `<TableFooter />` component did not break or change
  - You can see a `<TableFooter />` on the Kubernetes Details page in the "Node Pools" section
